LICENSE-MIT

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) [year] [fullname]
```

Comment:
All looks correct to me except that you missed the `year` and `copyright holder` in the MIT. For example

```
Copyright (c) 2024–Present MBON Contributers
```

Up to you, but listing everybody individually creates an annoying maintance problem should other people contribute.

Reply:
Thanks for catching that! I don't think I would have noticed it!